apps/hermes/server/src/state/aggregate.rs

```diff
@@ -169,6 +171,42 @@ pub enum Update {
     AccumulatorMessages(AccumulatorMessages),
 }
 
+#[derive(Debug, PartialEq)]
+pub struct PriceFeedTwapUpdate {
+    pub price_feed_id: PriceIdentifier,
+    pub cumulative_price: Price,
+    pub cumulative_num_down_slots: u64,
+    pub slot: Option<Slot>,
+    pub received_at: Option<UnixTimestamp>,
+    pub prev_publish_time: Option<UnixTimestamp>,
+}
+impl From<(&TwapMessage, Slot, UnixTimestamp)> for PriceFeedTwapUpdate {
```

Comment:
It is typically expected in `From` trait to consume the given objects (i.e. no reference). I also generally think these sort of conversions (passing a tuple) are non-obvious and for non-obvious conversions is better to have an explicit function.

Reply:
Good note, will keep that in mind for next time! Also, turns out this struct isn't even needed (leftover from a previous iteration), removed it